src/scss/includes/components/panel.scss

```diff
@@ -83,41 +78,17 @@
 
     &.minimized {
       height: 50px;
-
-      .minimizedTitle {
-        text-align: center;
+      .panel-drawer {
+        min-height: 50px;
       }
     }
 
-    &-content {
-      max-height: none;
-      height: calc(100% - 50px);
-      overflow: hidden;
-    }
-
-    .panel-resizeHandle {
-      pointer-events: all;
-      -moz-user-select: none;
-      -webkit-user-select: none;
-      -ms-user-select: none;
-      user-select: none;
-      min-height: 50px;
-      width: 100%;
-      cursor: -webkit-grab;
-      cursor: grab;
-      position: relative;
-
-      &:before {
-        content: '';
-        position: absolute;
-        top: 8px;
-        left: 50%;
-        width: 40px;
-        height: 5px;
-        transform: translateX(-50%);
-        border-radius: 2.5px;
-        background-color: #e0e1e6;
-      }
+    .panel-handle {
```

Comment:
What is the point of making it a real element instead of pseudo? It was the same amout of CSS but less DOM.

Reply:
The minimized title can be "pushed" from the drawer by making it a real elem

Reply:
Meh, not really convinced.